<launch>
  <node pkg="mujoco_tactile_sensor_plugin" type="MarkerPublisher" name="marker_publisher"
        output="screen">
    <remap from="tactile_sensor" to="/mujoco/tactile_sensor"/>
    <remap from="marker_arr" to="/mujoco/tactile_sensor/marker_arr"/>
  </node>

  <node pkg="rviz" type="rviz" name="$(anon rviz)"
        args="-d $(find mujoco_tactile_sensor_plugin)/launch/display.rviz" />
</launch>
